lang/en.json

```
@@ -151,6 +151,11 @@
         "CustomVariables": "Custom variables",
         "CustomizeTracking": "Customize Tracking",
         "CustomizeTrackingTeaser": "Want to configure the Matomo tracking for this container? Here is a list of existing Matomo configurations in this container:",
+        "DashboardHelp": "<p><strong>How does the \"Tag Manager > Dashboard\" page help me?</strong></p><p>The Tag Manager Dashboard provides an instant overview of both the names and quantity of tags, triggers, variables and versions of container configurations that have been created in your Matomo Tag Manager.</p><p>Within the Tag Manager Dashboard, you also can edit or create new tags, triggers, variables and versions.</p><p>Learn more about this in the %1$sTag Manager Dashboard Guide%2$s.</p>",
```

@snake14 We do not include HTML here you can use something like this.
```suggestion
        "DashboardHelp": "%1$s%2$sHow does the \"Tag Manager > Dashboard\" page help me?%3$s%4$s%1$sThe Tag Manager Dashboard provides an instant overview of both the names and quantity of tags, triggers, variables and versions of container configurations that have been created in your Matomo Tag Manager.%4$s%1$sWithin the Tag Manager Dashboard, you also can edit or create new tags, triggers, variables and versions.%4$s%1$sLearn more about this in the %5$sTag Manager Dashboard Guide%6$s.%1$s",
```

---

Or consider creating one entry per paragraph to separate them. It would lead to better results with the translations also 👍🏽

---

Yes makes more sense `DashboardHelpPara1`, `DashboardHelpPara2` and so on

---

Yeah. I used placeholders for the anchors, but the [developer documentation](https://developer.matomo.org/guides/translations#best-practices-for-new-translation-keys) said that it's alright to use basic tags, so I didn't bother splitting them out. I can break up the translations, if you think I should.

---

@justinvelluppillai We generally don't include HTML tags right ?
Or this one is okay to pass ?

---

We do use some, sometimes. In this case I think just creating three translation strings is worthwhile, and serves the purpose of making translations easier to maintain and removing a few tags.

---

`DashboardHelp1`, `DashboardHelp2`, `DashboardHelp3` is quite common

---

@snake14 Just split the longer ones into sub paras

---

Sounds good. I'll try to do that Monday morning. I'm trying to see if I can get the rest of the issue in another PR before I sign off for the day.

---

Thanks @snake14, sorry for the inconvenience we can explain/discuss why next week!